service/controller/v19/adapter/guest_route_tables.go

```diff
@@ -1,15 +1,23 @@
 package adapter
 
 import (
+	"fmt"
+
 	"github.com/giantswarm/microerror"
 
 	"github.com/giantswarm/aws-operator/service/controller/v19/key"
 )
 
+type RouteTableName struct {
+	ResourceName        string
+	InternalName        string
```

Comment:
Can we call it `TagName` so it is crystal clear what it actually is?

Reply:
Yeah, changed